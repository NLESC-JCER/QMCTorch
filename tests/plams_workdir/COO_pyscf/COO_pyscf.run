#!/bin/sh

$ADFBIN/adf <"COO_pyscf.in"

