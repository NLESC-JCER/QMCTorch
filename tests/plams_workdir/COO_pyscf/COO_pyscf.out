Error while handling the Basis key.
Error: Could not find matching basis file for O

Input found:

units
  length Bohr
end

XC
  HartreeFock
end

atoms
      1         C      0.000000      0.000000      0.000000 
      2         O      0.000000      0.000000      2.190000 
      3         O      0.000000      0.000000     -2.190000 
end

basis
  core None
  type PYSCF
end

symmetry nosym

totalenergy




